src/Output/Output_DumpData.cpp

```diff
@@ -126,22 +130,22 @@ void Output_DumpData( const int Stage )
 
       if ( OPT__OUTPUT_BASE )
       {
-         sprintf( FileName_Part, "%s", "Base" );
+         sprintf( FileName_Part+str_offset, "%s", "Base" );
```

Comment:
How about simplifying the code as follows, so `str_offset` is no longer needed?
```suggestion
         strcat( FileName_Part, "Base" );
```